src/AstTranslator.cpp

```diff
@@ -168,6 +168,7 @@ std::vector<IODirectives> AstTranslator::getOutputIODirectives(
             IODirectives ioDirectives;
             if (dynamic_cast<const AstPrintSize*>(current) != nullptr) {
                 ioDirectives.setIOType("stdoutprintsize");
+                ioDirectives.set("summary", "true");
```

Comment:
An IOType of "stdoutprintsize" is a unique identifier for a printsize, so the summary directive is redundant.